product-service/serverless.ts

```diff
@@ -25,8 +25,13 @@ const serverlessConfiguration: Serverless = {
     },
     environment: {
       AWS_NODEJS_CONNECTION_REUSE_ENABLED: '1',
+      PG_HOST: 'task4-instance.c4bpr1tbr0zy.eu-west-1.rds.amazonaws.com',
```

Comment:
don't push creds to the repo. Please use environment variables

Reply:
Thanks, got it, I though it's fine to store here the public variables.